CONTRIBUTION.md

```diff
@@ -0,0 +1,83 @@
+# Contribution To The Project
+
+Pull Requests and Issues have four possible Labels:
+* WIP - Work In Progress, it is not ready for review/merge
+* Functional - Proposed change changes the test behaviour
+* Metadata - Only Metadata is about to change
+* Scope - Big scope, a commit changes multiple test scenarios
+
+Rules:
+* A commit introduces one logical unit and it should not alter multiple testcases
+* Do not merge your own Pull Request
+
+
+Workflow:
+* Create a fork
+* Clone a fork
+* Add remote to your fork, call it upstream
+```
+$ git remote add upstream git@github.com:RedHat-SP-Security/tests.git
```

Comment:
I would use https://github.com/RedHat-SP-Security/tests.git here, to make it slightly harder to push something there by accident.

Reply:
I would prefer public key authentication. I've mention it https://github.com/RedHat-SP-Security/tests/pull/7/commits/722895b95098fb6ca7486e658a6954465c703ce7#diff-0c8a93902edcf8795d61d5e98dc43d4ecd8645f4a924e67c080a70f364edc78eR14